internal/provider/kubernetes/test/utils.go

```diff
@@ -0,0 +1,145 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package test
```

Comment:
None of the types, func, etc. are specific to testing. Why not have this as a `utils`, `resources`, etc. pkg?

Reply:
these just return sample objects that are only being used in _tests